Postman Code Generation - change arguments names (#1719)
demisto/demisto-sdk: CHANGELOG.md

* The **format** command will now not remove the `defaultRows` key from incident, indicator and generic fields with `type: grid`.
* Fixed an issue with the **validate** command when a layoutscontainer did not have the `fromversion` field set.
* Added `skipVerify` argument in **upload** command to skip pack signature verification.

> * The **postman-codegen** will not have duplicates arguments' name, it renames them to the minimum distinguishing shared path between it and other arguments with the same name.

Reviewer: ```suggestion
* The **postman-codegen** will not have duplicates arguments' name. It will rename them to the minimum distinguished shared path for each of them.
```
